usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Usecase WebAPI
+ Scenario: Device & Hardware/BrowserState Test
+    When launch "usecase-webapi-xwalk-tests"
+     And I go to "/samples/BrowserState/index.html"
+     And I open airplane mode
+     And I wait 10 seconds
+     And I execute command "adb shell am force-stop com.android.settings"
+    Then I should see "online status : false"
+    Then I should not see "N/A"
+     And I close airplane mode
```

Comment:
Here we have a risk that if one of above steps fails, the airplane mode is open which would impact other tests.